### PR TITLE
initial changes to menuitem

### PR DESCRIPTION
**Pull request by milenapribic**

### CODE REVIEW

File: client/components/MenuItem/stories/index.js [hunk 1]
```diff
@@ -5,8 +5,6 @@ import { storiesOf } from '@kadira/storybook';
 storiesOf('MenuItem', module)
   .add('default view', () => {
     return (
-      <div className="menuItem">
-        <MenuItem />
-      </div>
+      <MenuItem href="/milena" name="Milena" isActive={true} />
```
Comment (chrisdhanaraj) [inline client/components/MenuItem/stories/index.js]: listen milena, the default view is not active. fix pls

**Comment by milenapribic** [comment]
style single menu item, add scss file for storybook, removed `{component} from import`

File: client/components/MenuItem/index.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
-import React, { Component } from 'react';
+import React from 'react';
+
+import reset from '../../utils/reset.scss';
+import menuItemStyles from './menuitem.scss';
+import { Link } from 'react-router';
+
+//what props do i need? link href / name of link / active or not
```
Comment (chrisdhanaraj) [inline client/components/MenuItem/index.js]: can you remove these comments please

File: client/components/MenuItem/index.js [hunk 3]
```diff
@@ -1,9 +1,29 @@
-import React, { Component } from 'react';
+import React from 'react';
+
+import reset from '../../utils/reset.scss';
```
Comment (chrisdhanaraj) [inline client/components/MenuItem/index.js]: if you're not using the variable for anything, go ahead and just `import '../../utils/reset.scss'`

File: client/components/MenuItem/menuitem.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.menu-item {
+  display: inline-block;
+  padding: 5px 15px 5px 5px;
+
+  font-family: 'Helvetica Neue';
+  font-weight: bold;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+  &.is-active{
```
Comment (chrisdhanaraj) [inline client/components/MenuItem/menuitem.scss]: add a new line between 

`color: #fff`

and 

`&.is-active {`

**Comment by milenapribic** [comment]
did it all thx

**Comment by chrisdhanaraj** [comment]
:+1:

**Pull request merged**